introduction/features.md

```diff
@@ -1,19 +1,22 @@
 ## Features
 
-* Fast
-* [Typo tolerance](/advanced_guides/typotolerance.md#typo-tolerance) (understands typo and spelling mistakes)
-* Return whole document
-* Crop, snippets, filters, highlight in search queries
-* Search inside chosen attributes
-* Highlighted results
-* Prefix-search
-* Configurable Stop-words
-* Configurable Synonyms
-* Asynchronous action on POST, PUT, and DELETE
-* RESTfull API
-* Supported Latin and Kanji [tokenizer](https://github.com/meilisearch/MeiliDB/blob/dc5c42821e1340e96cb90a3da472264624a26326/meilidb-tokenizer/src/lib.rs) 
-* Bucket sort with 6 default rankings criteria
-* Custom criteria (ranking, distinct, ...) in schema definition
-<!-- Schemaless possibility -->
+Our feature list with extended explanations and links to the documentation.
 
 
+* **Instant Search** (answers < 50ms): Priority on fast answers for smooth search experience.
+* **Search as you type** (*prefix search*): Results are shown as you type. To achieve this, we use a [prefix-search](/advanced_guides/prefix.html#prefix-search).
```

Comment:
```suggestion
* **Search as you type** (*prefix search*): Results are updated on each keystroke. We use a [prefix-search](/advanced_guides/prefix.html#prefix-search) for the best results.
```

🤷‍♀

Reply:
I wanted to highlight that the instant search was made possible by our use of a prefix-search. And not that the prefix-search made for best results.